convertapi/client.py

```diff
@@ -50,6 +51,9 @@ def handle_response(self, r):
 			except ValueError:
 				raise e
 
+		if r.content == b'':
+			raise AsyncConversionInProgress
+
```

Comment:
This is to solve a problem mentioned in issue #21: while the conversion is in progress, it seems the response from the server is a 202 Accepted with a blank response.

In the tests, the very first request after the async conversion has a status code 200 and blank content. The second request would have a status 202, as well as the subsequent requests until a 200 response with the expected result.

That's why I'm not testing for a 202 status code.

Reply:
Notice this would raise a JSONDecodeError for any get request that doesn't have a JSON response, and I don't want to fail with an AsyncConversionInProgress error for any request. This way, I'm returning None, and only on the async_poll function I will check for the empty response and raise the exception accordingly.